template/partials/class.tmpl.partial

```diff
@@ -1,61 +1,62 @@
 {{#isClass}}
 
-<div>
-  <h1 id="{{id}}" data-uid="{{uid}}" class="text-break" style="display: inline-block;">
-    {{name.0.value}}
-    {{#sourceurl}}<a class="header-action link-secondary" title="View source" href="{{sourceurl}}"><i class="bi bi-code-slash"></i></a>{{/sourceurl}}
-  </h1>
-  {{#oe.operatorType}}
-  <p style="display: inline; text-transform: capitalize"> 
-    {{oe.operatorType}} Operator
-  </p>
-  {{/oe.operatorType}}
-</div>
-
-{{#oe.device}}
+{{#oe.operator.configureDevice}}
 
 <div class = "alert alert-danger">
-  This operator is a device operator. If you are using Open Ephys hardware, use an aggregate operator that corresponds to your particular headstage(s) instead. Aggregate operators confer the following benefits:
+  This is a device configuration operator. If you are using Open Ephys hardware, use an aggregate configuration operator that corresponds to your particular headstage(s) instead. Aggregate operators confer the following benefits:
   <ul style = "margin-bottom: 0">
-    <li>The <code>address</code> properties of aggregate operators are automatically configured whereas a device operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
-    <li>All devices that belong to a hub (which is a collection of devices i.e. a headstage or breakout board) are automatically included, excluding the possibility of omitting configuration of a device that belongs to a hub. This improves ease of using Open Ephys hardware.</li>
-    <li>The workflow is less cluttered with configuration operators as one aggregate operator can correspond to multiple device operators. This improves workflow readability.</li>
+    <li>The <code>address</code> properties of aggregate configuration operators are automatically configured whereas a device configuration operators requires manual configuration of its <code>address</code> property. This improves ease of using Open Ephys hardware.</li>
```

Comment:
This text is also present in the warning on the `device-configure.md` page; in my opinion these two sections should not be carbon copies of each other. It should remain here as-is (minus some potential changes [here](https://github.com/open-ephys/onix1-bonsai-docs/pull/26#discussion_r1721731208)), and the `device-configure.md` should be a shorter version since it is encapsulated in a Caution tag already

Reply:
I agree. I just wasn't sure how to make these carbon copies because the template is written in HTML and the admonition in device-configure is in markdown. It might be simple to include it, but iirc there were linkcheck errors. I will look into it again.